packages/wrangler/src/__tests__/cloudchamber/apply.test.ts

```diff
@@ -130,12 +130,12 @@ describe("cloudchamber apply", () => {
 			│   instances = 3
 			│   scheduling_policy = \\"default\\"
 			│
-			│   [containers.constraints]
-			│   tier = 2
+			│     [containers.constraints]
+			│     tier = 2
 			│
-			│   [containers.configuration]
-			│   image = \\"./Dockerfile\\"
-			│   instance_type = \\"dev\\"
+			│     [containers.configuration]
+			│     image = \\"./Dockerfile\\"
+			│     instance_type = \\"dev\\"
```

Comment:
The extra indentation here and throughout is because the [TOML library we use](https://www.npmjs.com/package/@iarna/toml) adds indentation to nested tables. This is, unfortunately, not configurable. We can propose a patch upstream to make this configurable, but it's also not really a big deal.